enhancements/update/openstack-cacerts.md

```diff
@@ -0,0 +1,208 @@
+---
+title: openstack-cacert
+authors:
+  - "@stephenfin"
+reviewers:
+  - "TBD"
+approvers:
+  - "TBD"
+api-approvers:
+  - "None"
+creation-date: 2025-03-19
+last-updated: 2025-03-20
+tracking-link:
+  - https://issues.redhat.com/browse/OSASINFRA-3729
+see-also:
+  - "None"
+replaces:
+  - "None"
+superseded-by:
+  - "None"
+---
+
+# OpenStack CA Cert migration
+
+## Summary
+
+This enhancement proposes an update strategy for migrating the location of the
+CA cert used for clusters deployed on OpenStack using self-signed certs.
+
+## Motivation
+
+OpenStack is frequently used as an on-prem cloud solution, and these
+deployments often use self-signed certificates. Certificates expire and need to
+be rotated when they do, however, there is currently no easy way to rotate
+these as they live in multiple places. In addition, the current mechanisms we
```

Comment:
Please can we list these places, ideally with links to user documentation about how to create and update them?